security/login_link.rst

```diff
@@ -654,3 +654,52 @@ user create this POST request (e.g. by clicking a button)::
             <button type="submit">Continue</button>
         </form>
     {% endblock %}
+
+Customize generated Login Link
+------------------------------
+
+.. versionadded:: 5.3
+
+    This feature was introduced in Symfony 5.3.
```

Comment:
The possibility to customize the login link was introduced in Symfony 5.3.

Reply:
Corrected